pkg/minikube/reason/exitcodes.go

```diff
@@ -78,7 +78,7 @@ const (
 	ExHostNotFound    = 35
 	ExHostUnsupported = 38
 	ExHostPermission  = 37
-	ExHostConfig      = 38
+	ExHostConfig      = 39
```

Comment:
how about instead of changing this to 39 
ExHostUnsupported = 36
that way they would be in order
```
	ExHostTimeout     = 32
	ExHostUsage       = 34
	ExHostNotFound    = 35
	ExHostUnsupported = 36
	ExHostPermission  = 37
	ExHostConfig      = 38
```